### PR TITLE
Update to vue 3

### PR DESCRIPTION
**Pull request by bdemers**
Merged changes from Okta mono repo: https://github.com/okta/samples-js-vue/tree/master/okta-hosted-login

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,47 +9,25 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "^4.9.2",
-    "@okta/okta-vue": "^3.1.0",
+    "@okta/okta-auth-js": "^4.6.1",
```
Comment (mraible) [inline package.json:12]: I'd update to the latest version: v4.9.2.

**Review by mraible — APPROVED** [review]

**Comment by mraible** [comment]
@bdemers Can you please rebase?

**Comment by mraible** [comment]
Will it be possible to still create a Vue 2 version with the CLI? I ask because once this is merged, it seems like OktaDev Schematics will be the only way to do it. https://devforum.okta.com/t/okta-integration-with-vue-js-2-0/14835/2

**Comment by bdemers** [comment]
Maybe we should create an `okta-vue2-sample` project, or qualify any Vue example with a version?
- `okta-vue2-sample`
- `okta-vue3-sample`

**Comment by mraible** [comment]
We might want to upgrade to [Okta Vue 5.0.0](https://github.com/okta/okta-vue/releases/tag/okta-vue-5.0.0) before merging since it was just released.

**Comment by mraible** [comment]
@bdemers Can you please rebase so we can merge dependabot updates?

**Pull request merged**